new-components/scanners/bandit/internal/transformer/transformer.go

```diff
@@ -135,7 +135,7 @@ func New(opts ...BanditTransformerOption) (*BanditTransformer, error) {
 	rawOutFilePath, err := env.GetOrDefault(
 		"BANDIT_RAW_OUT_FILE_PATH",
 		"bandit.json",
-		env.WithDefaultOnError(true),
+		env.WithDefaultOnError(false),
```

Comment:
Are we sure we're not ok with having the default if this is missing? 🤔

Reply:
we really want this to fail if there is no input bandit file